pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch_handlerchain.go

```diff
@@ -37,6 +37,11 @@ func BuildHandlerChain(genericConfig *genericapiserver.Config, kubeInformers inf
 		extraPostStartHooks[name] = fn
 	}
 
+	extraPostStartHooks["openshift.io-catransportupdater"] = func(context genericapiserver.PostStartHookContext) error {
+		go caTransportUpdater.Run(context.StopCh)
+		return nil
+	}
+
 	return func(apiHandler http.Handler, genericConfig *genericapiserver.Config) http.Handler {
 			// Machinery that let's use discover the Web Console Public URL
 			accessor := newWebConsolePublicURLAccessor(genericConfig.LoopbackClientConfig)
```

Comment:
This needs to be moved out of the handler.  I am surprised it even works at all.  This may explain the weird behavior you were seeing locally.